.devcontainer/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
         curl jq \
         vim-nox neovim python3-pynvim \
         make \
+        rename \
```

Comment:
One other small change from #313 - adds the `rename` utility to the devcontainer.